src/SFA.DAS.ApprenticeCommitments.Web/Startup/NLogStartup.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NLog.Web;
+using System;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Startup
+{
+    public static class NLogStartup
+    {
+        public static void ConfigureNLog()
+        {
+            try
+            {
+                var environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+                var configFileName = environment == "Development" ? "nlog.Development.config" : "nlog.config";
+                NLogBuilder.ConfigureNLog(configFileName);
```

Comment:
like this

Reply:
I don't think we need the javascipt, it was part of the default template that I forgot to remove.  I'll get rid of it.